File: theFinale/Models/Vault.cs
namespace theFinale.Models
{
  public class Vault
  {
    public int id { get; set; }
    public string creatorId { get; set; }
    public string name { get; set; }
    public string description { get; set; }
    public string img { get; set; }
    public bool isPrivate { get; set; }

    // virtuals

    public Account creator { get; set; }
  }
}

File: theFinale.client/src/models/Vault.js
export class Vault {
    constructor(data) {
        this.id = data.id
        this.creatorId = data.creatorId
        this.name = data.name
        this.description = data.description
        this.img = data.img
        this.isPrivate = data.isPrivate
        // virtual
        this.creator = data.creator
    }
}